tools/libraryowners/main.go

```diff
@@ -58,19 +58,40 @@ func newLibraryOwners() (map[string]string, error) {
 	return libraryOwnersContent, nil
 }
 
-func validate(libraryOwner map[string]string, goMod *modfile.File) error {
+func validateNewLibs(libOwners map[string]string, goMod *modfile.File) error {
+	var addedLibs []string
 	for _, library := range goMod.Require {
 		if library.Indirect {
 			continue
 		}
-
-		if val, ok := libraryOwner[library.Mod.Path]; !ok {
-			return fmt.Errorf("'%s' is not inside '%s'. Please, remove this dependency from '%s'", library.Mod.Path, libraryOwnersPath, goModpath)
-		} else if val == "" {
-			return fmt.Errorf("'%s' doesn't have a owner. Please, add a owner to %s in '%s", library.Mod.Path, library.Mod.Path, libraryOwnersPath)
+		if val, ok := libOwners[library.Mod.Path]; !ok || val == "" {
+			addedLibs = append(addedLibs, library.Mod.Path)
```

Comment:
@andreaangiolillo FYI I added 3 libs and this was only giving me an error on first find so improving errors here to report all new libs